src/index.js

```diff
@@ -157,16 +160,28 @@ export default function nodeResolve ( options = {} ) {
 				importee,
 				Object.assign( resolveOptions, customResolveOptions )
 			)
-				.catch(() => false)
+				.catch(() => FALSE)
```

Comment:
Minor suggestion: Looking at the following `.then`, the `FALSE` case basically reduces to returning `null` without any further code being executed. If you move this `.catch` to the end and rewrite it as `.catch(() => null)`, it has nearly the same effect but you can get rid of `FALSE` and the associated checks.

Reply:
thx, optimized in the latest commit.